packages/vue3/src/components/Divider/Divider.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
+import type { Orientation, Style } from './types';
+
+withDefaults(
+  defineProps<{
+    orientation?: Orientation
+    borderStyle?: Style
+  }>(),
+  {
+    orientation: 'horizontal',
+    borderStyle: 'solid',
+  },
+);
+</script>
+
+<template>
+  <div :class="{ [orientation]: true, [borderStyle]: true }" />
+</template>
+
+<style scoped lang="scss">
+.horizontal {
+  border-width: 0.5px;
+  border-color: var(--gray-400);
+  width: 90%;
```

Comment:
Let's make it so that it takes the entire width (.horizontal) and entire height (.vertical)